builder/vsphere/supervisor/step_import_image.go

```diff
@@ -0,0 +1,367 @@
+// Copyright (c) HashiCorp, Inc.
+// SPDX-License-Identifier: MPL-2.0
+
+//go:generate packer-sdc struct-markdown
+//go:generate packer-sdc mapstructure-to-hcl2 -type ImportImageConfig
+
+package supervisor
+
+import (
+	"context"
+	"fmt"
+	"strings"
+	"time"
+
+	"github.com/google/uuid"
+	"github.com/hashicorp/packer-plugin-sdk/multistep"
+	imgregv1 "github.com/vmware-tanzu/image-registry-operator-api/api/v1alpha1"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/fields"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+const (
+	ImportTargetKind       = "ContentLibrary"
+	ImportTargetAPIVersion = "imageregistry.vmware.com/v1alpha1"
+
+	DefaultWatchImageImportTimeoutSec = 600
+
+	ImportRequestDefaultNamePrefix    = "packer-vsphere-supervisor-import-req-"
+	StateKeyImageImportRequestCreated = "item_import_req_created"
+	StateKeyCleanImportedImage        = "clean_imported_image"
+	StateKeyImportedImageName         = "imported_image_name"
+
+	importOVFFeatureNotEnabledMsg = "WCP_VMImageService_ImportOVF feature is not enabled"
+)
+
+var IsWatchingImageImport bool
+
+type ImportImageConfig struct {
+	// The remote URL where the to-be-imported image is hosted.
+	ImportSourceURL string `mapstructure:"import_source_url"`
+	// The SSL certificate of the remote HTTP server that hosts the to-be-imported image.
+	ImportSourceSSLCertificate string `mapstructure:"import_source_ssl_certificate"`
+
+	// Name of a writable & import-allowed ContentLibrary resource in the namespace where the image will be imported.
```

Comment:
```suggestion
	// Name of a writable an import-allowed contentl ibrary resource in the namespace where the image will be imported.
```

Reply:
I presume you mean to change `&` to `and`, updated it.